docs/reference/modules/remote-clusters.asciidoc

```diff
@@ -1,29 +1,40 @@
-[[modules-remote-clusters]]
+[[remote-clusters]]
 == Remote clusters
-
 You can connect a local cluster to other {es} clusters, known as _remote
-clusters_. Once connected, you can search remote clusters using
-<<modules-cross-cluster-search,{ccs}>>. You can also sync data between clusters
-using <<xpack-ccr,{ccr}>>.
-
-To register a remote cluster, connect the local cluster to nodes in the
-remote cluster using one of two connection modes:
-
-* <<sniff-mode,Sniff mode>>
-* <<proxy-mode,Proxy mode>>
-
-Your local cluster uses the <<modules-network,transport layer>> to establish
-communication with remote clusters. The coordinating nodes in the local cluster
-establish <<long-lived-connections,long-lived>> TCP connections with specific
-nodes in the remote cluster. {es} requires these connections to remain open,
-even if the connections are idle for an extended period.
-
-You can use the <<cluster-remote-info, remote cluster info API>> to get
-information about registered remote clusters.
+clusters_. Remote clusters can be located in different datacenters or 
+geographic regions, and contain indices or data streams that can be replicated
+with {ccr} or searched by a local cluster using {ccs}.
+
+With <<xpack-ccr,{ccr}>>, you ingest data to an index on a remote cluster. This 
+_leader_ index is replicated to one or more read-only _follower_ indices on your local cluster. Creating a multi-cluster architecture with {ccr} enables you to 
+configure disaster recovery, bring data closer to your users, or establish a 
+centralized reporting cluster to process reports locally.
+
+<<modules-cross-cluster-search,{ccs-cap}>> enables you to run a search request 
+against one or more remote clusters. This capability provides each region
+with a global view of all clusters, allowing you to send a search request from
+a local cluster and return results from all connected remote clusters. 
+
+Enabling and configuring security is important on both local and remote
+clusters. When connecting a local cluster to remote clusters, an {es} 
+superuser (such as the `elastic` user) gains total read access to the remote 
+clusters. Furthermore, a local user with administrative privileges at the 
+operating system level can alter `elasticsearch.yml`. A malicious user with
+these privileges has potential access to private keys that could give them the 
+ability to take over a remote cluster. To use {ccr} and {ccs} safely, 
```

Comment:
It still does not read straightforward to me because it seems to suggest "alter elasticsearch.yml" is a reason for having "potential access to private keys". But these two things are separate. Maybe it's just me? I'd suggest the following change:

```suggestion
clusters. Furthermore, a local administrator at the operating system level with sufficient access 
to Elasticsearch config files and private keys can potentially take over a remote cluster. To use {ccr} and {ccs} safely, 
```

Also, I wonder whether we should take this sentence out from here because Enabing security at the Elasticsearch level does not help mitigating this threat. I think we can move it to the end of this paragraph after we finishing talking about ES level security first, something like:

> Enabling and configuring security is important on both local and remote clusters. When connecting a local cluster to remote clusters, an Elasticsearch superuser (such as the elastic user) gains total read access to the remote clusters. To use cross-cluster replication and cross-cluster search safely, enable security on all connected clusters and configure Transport Layer Security (TLS) on at least the transport level on every node. Furthermore, a local administrator at the operating system level with sufficient access to Elasticsearch config files and private keys can potentially take over a remote cluster. **Therefore you should treat the operating system level security for both clusters with equal importance.**

I added the sentence in bold as a very generic advice for OS level security. Otherwise, it feels that we raise an issue without offering any recommendation. Please feel free to paraphrase.

Reply:
I split this information into two paragraphs:

> Enabling and configuring security is important on both local and remote
> clusters. When connecting a local cluster to remote clusters, an {es} superuser
> (such as the `elastic` user) on the local cluster gains total read access to the
> remote clusters. To use {ccr} and {ccs} safely, 
> <<remote-clusters-security,enable security>> on all connected clusters 
> and configure Transport Layer Security (TLS) on at least the transport level on
> every node.
> 
> Furthermore, a local administrator at the operating system level
> with sufficient access to {es} configuration files and private keys can
> potentially take over a remote cluster. Ensure that your security strategy
> includes securing local _and_ remote clusters at the operating system level.